examples/test_iam_policy_management_v1_examples.py

```diff
@@ -1429,6 +1434,388 @@ def test_delete_action_control_template_example(self):
         except ApiException as e:
             pytest.fail(str(e))
 
+    # @needscredentials
+    def test_create_role_template_example(self):
+        """
+        create_role_template request example
+        """
+        try:
+            print('\ncreate_role_template() result:')
+
+            # begin-create_role_template
+
+            response = iam_policy_management_service.create_role_template(
+                name='SDKRoleTemplateExample',
+                account_id=example_account_id,
+            )
+            role_template = response.get_result()
+
+            global example_role_template_id
+            example_role_template_id = role_template['id']
+            global example_role_template_version
+            example_role_template_version = role_template['version']
+
+            print(json.dumps(role_template, indent=2))
+
+            # end-create_role_template
+
+        except ApiException as e:
+            pytest.fail(str(e))
+
+    # @needscredentials
+    def test_get_role_template_example(self):
+        """
+        get_role_template request example
+        """
+        try:
+            print('\nget_role_template() result:')
+
+            # begin-get_role_template
+
+            response = iam_policy_management_service.get_role_template(
+                role_template_id=example_role_template_id,
+            )
+            role_template = response.get_result()
+
+            global example_role_template_etag
+            example_role_template_etag = response.get_headers().get("Etag")
+
+            print(json.dumps(role_template, indent=2))
+
+            # end-get_role_template
```

Comment:
Please move any post-global-variable-assignment out of the actual example block. It's not part of the example but our logic to make the file runnable. For example:

```suggestion
            print(json.dumps(role_template, indent=2))

            # end-get_role_template
            
            global example_role_template_etag
            example_role_template_etag = response.get_headers().get("Etag")
```